machines/config_machines.xml

```diff
@@ -406,6 +406,103 @@ This allows using a different mpirun command to launch unit tests
+    <BASELINE_ROOT>$ENV{CESMDATAROOT}/cesm_baselines</BASELINE_ROOT>
+    <CCSM_CPRNC>$ENV{CESMDATAROOT}/cprnc/cprnc</CCSM_CPRNC>
+    <GMAKE_J>16</GMAKE_J>
+    <BATCH_SYSTEM>pbs</BATCH_SYSTEM>
+    <SUPPORTED_BY>cseg</SUPPORTED_BY>
+    <MAX_TASKS_PER_NODE>128</MAX_TASKS_PER_NODE>
+    <MAX_MPITASKS_PER_NODE>128</MAX_MPITASKS_PER_NODE>
+    <PROJECT_REQUIRED>TRUE</PROJECT_REQUIRED>
+    <mpirun mpilib="default">
+      <executable>mpibind</executable>
+      <arguments>
+        <arg name="label"> --label</arg>
+        <arg name="buffer"> --line-buffer</arg>
+        <arg name="separator"> -- </arg>
+      </arguments>
+    </mpirun>
+    <module_system type="module" allow_error="true">
+      <init_path lang="perl">$ENV{LMOD_ROOT}/lmod/init/perl</init_path>
+      <init_path lang="python">$ENV{LMOD_ROOT}/lmod/init/env_modules_python.py</init_path>
+      <init_path lang="sh">$ENV{LMOD_ROOT}/lmod/init/sh</init_path>
+      <init_path lang="csh">$ENV{LMOD_ROOT}/lmod/init/csh</init_path>
+      <cmd_path lang="perl">$ENV{LMOD_ROOT}/lmod/libexec/lmod perl</cmd_path>
+      <cmd_path lang="python">$ENV{LMOD_ROOT}/lmod/libexec/lmod python</cmd_path>
+      <cmd_path lang="sh">module</cmd_path>
+      <cmd_path lang="csh">module</cmd_path>
+      <modules>
+        <command name="load">ncarenv/24.12</command>
+        <command name="purge"/>
+	      <command name="load">craype</command>
+      </modules>
+      <modules compiler="intel">
+        <command name="load">intel/2024.2.1</command>
+	      <command name="load">mkl</command>
+        <command name="load">kokkos</command>
+      </modules>
+      <modules compiler="intel-oneapi">
+        <command name="load">intel-oneapi/2024.2.1</command>
+	      <command name="load">mkl</command>
+      </modules>
+      <modules compiler="cray">
+        <command name="load">cce/17.0.1</command>
+        <command name="load">cray-libsci/24.03.0</command>
+      </modules>
+      <modules compiler="gnu.*">
+        <command name="load">gcc/12.4.0</command>
+        <command name="load">cray-libsci/24.03.0</command>
+      </modules>
+      <modules compiler="gnugpu">
+        <command name="load">cuda/12.3.2</command>
+        <command name="load">kokkos</command>
+      </modules>
+      <modules compiler="nvhpc">
+        <command name="load">nvhpc/24.11</command>
+      </modules>
+      <modules>
+	      <command name="load">ncarcompilers/1.0.0</command>
+        <command name="load">cmake</command>
+      </modules>
+      <modules mpilib="mpich">
+	      <command name="load">cray-mpich/8.1.29</command>
+      </modules>
+      <modules mpilib="mpi-serial">
+        <command name="load">mpi-serial/2.5.0</command>
+        <command name="load">netcdf/4.9.2</command>
+      </modules>
+      <modules mpilib="!mpi-serial">
+        <command name="load">netcdf-mpi/4.9.2</command>
+        <command name="load">parallel-netcdf/1.14.0</command>
+      </modules>
+      <modules DEBUG="FALSE">
```

Comment:
```suggestion
      <modules>
```

Reply:
I think this needs to be loaded whether debug is selected or not.  My debug case failed on PIO because the module parellelio module didn't load and it couldn't detect that PIO was installed on the machine.  It worked fine if I removed it.  Otherwise all the rest is good to go.

Reply:
Thanks @jtruesdal . I thought there was a specific debug version of PIO on Derecho but it was not the case. Thus I have committed your suggested changes and it should be good to go now. I did not have permissions to merge PR into your repo so you can do it whenever you have time.